server/api_setup.py

```diff
@@ -1,17 +1,35 @@
 """This module is an API set up
 SHOULD WORK WITH YELP API"""
+
 import os
 import random
 import requests
 import sys
 from dotenv import find_dotenv, load_dotenv
 
+print(find_dotenv())
```

Comment:
Maybe remove this print statement

Reply:
I removed it.

Reply:
Great! go ahead and merge. I have approved it.